src/include/denc.h

```diff
@@ -170,9 +170,9 @@ struct denc_traits {
 #define WRITE_RAW_DENC(type)						\
   template<>								\
   struct denc_traits<type> {						\
-    enum { supported = 2 };						\
-    enum { featured = false };						\
```

Comment:
Run this past Sage. From the comments it looks like he did this to prevent some sort of clash with encode/decode, so don't remove it without his imprimatur.